source/frontend/src/hooks/useBcAssessmentLayer.ts

```diff
@@ -100,6 +100,7 @@ export const useBcAssessmentLayer = (
       }
       let folioId = legalDescriptionResponse?.features[0]?.properties?.FOLIO_ID;
       let rollNumber = legalDescriptionResponse?.features[0]?.properties?.ROLL_NUMBER;
+      let jurisdictionCode = legalDescriptionResponse?.features[0]?.properties?.JURISDICTION_CODE;
```

Comment:
this is the fix for psp-6404, which combines ensuring the CQL is being generated with an AND instead of an OR and also including a restriction by jurisidiction when looking for matches.